src/docs/arch-compact.dox

```diff
@@ -62,4 +69,21 @@ track current progress. Some of the verbose logs for compaction can be activated
 verbose logs, WT also records statistics about compact progress that can be used to debug issues
 with the compaction process.
 
+# Background compaction #
+
+Background compaction is a WiredTiger service that runs compaction autonomously on all the eligible files
+present in the database while being the least disruptive as possible. Every file is eligible unless:
+
+    - it belongs to a tiered table or,
+    - is part of the exclude list that can be configured when enabling background compaction.
+
+To limit disruption, background compaction reduces the number of checkpoints by skipping the first one
+described in the section above. Furthermore, background compaction gathers statistics over time to
+keep track of tables where compaction is successful. Using those statistics, the service focuses on
+tables where compaction is likely to reclaim space and avoids unnecessary attempts.
+
+Background compaction can be enabled, configured and disabled through the same API \c WT_SESSION::compact.
```

Comment:
np:
- comma after configured 
- ...as soon as possible and when re-enabled, it starts..."

Reply:
Thank you, see 0b119b18000615d5915fd895b70ca723217c6b41.